src/config/config_upgrade.c

```diff
@@ -15,17 +15,8 @@
 int
 __wt_config_upgrade(WT_SESSION_IMPL *session, WT_ITEM *buf)
```

Comment:
We had a chat with @agorrod and his suggestion is to remove this function entirely. It's unlikely we'll need to symlink API options to each other in the future.

Reply:
Removed this and references to this